ci/conda-recipe/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = environ.get('GIT_DESCRIBE_TAG', "0.dirty").replace('-', '.') %}
```

Comment:
Does this file belong in `ci`? I think there might be a better place for it in something like `build` or `packaging`.

Reply:
well, there is `ci/benchmarks` directory already which have the same or even less level of correlation with CI/CD as conda-recipe. I'll put it into `scripts/` - like in omniscidb